### ISSUE TITLE
AttributeError when using 'cupyx.scipy.signal.butter'

### ISSUE THREAD

**Issue by BWang15** [cupy/cupy#7915]
### Description

I was trying to design a lowpass filter using `butter` from `cupyx.scipy.signal`. But it gave me an error saying `AttributeError: module 'cupyx.scipy.signal' has no attribute 'butter'`. 
My Cupy was successfully installed via pip and I executed `cupy.fft` before the `butter` without any error. Did I miss anything when I install Cupy?

### To Reproduce

```py
# The import statement does not cause any error
from cupyx.scipy import signal

b, a = signal.butter(3, 35, fs=1024)
# AttributeError: module 'cupyx.scipy.signal' has no attribute 'butter'
```

### Installation

Source (`pip install cupy`)

### Environment

```
OS                           : Linux-5.15.0-84-generic-x86_64-with-glibc2.35
Python Version               : 3.10.12
CuPy Version                 : 12.2.0
CuPy Platform                : NVIDIA CUDA
NumPy Version                : 1.25.2
SciPy Version                : 1.11.1
Cython Build Version         : 0.29.36
Cython Runtime Version       : None
CUDA Root                    : /usr/local/cuda
nvcc PATH                    : /usr/local/cuda/bin/nvcc
CUDA Build Version           : 11080
CUDA Driver Version          : 12020
CUDA Runtime Version         : 11080
cuBLAS Version               : (available)
cuFFT Version                : 10900
cuRAND Version               : 10300
cuSOLVER Version             : (11, 4, 1)
cuSPARSE Version             : (available)
NVRTC Version                : (11, 8)
Thrust Version               : 101501
CUB Build Version            : 101501
Jitify Build Version         : <unknown>
cuDNN Build Version          : 8801
cuDNN Version                : 8901
NCCL Build Version           : None
NCCL Runtime Version         : None
cuTENSOR Version             : None
cuSPARSELt Build Version     : None
Device 0 Name                : NVIDIA GeForce RTX 3090
Device 0 Compute Capability  : 86
Device 0 PCI Bus ID          : 0000:3B:00.0

```

**Comment by kmaehashi** [comment]
`cupyx.scipy.signal.butter` is available in CuPy v13.0.0b1 or later (#7591). Please try out pre-releases 😃
https://github.com/cupy/cupy/?tab=readme-ov-file#installation

**Comment by BWang15** [comment]
Oh, thank you. Did not notice the pre-releases.
